subsys/power/device.c

```diff
@@ -40,6 +40,11 @@ static const char *const core_devices[] = {
 static const char *const core_devices[] = {
```

Comment:
In the commit msg, the "subsys/" is superfluous and can be omitted, imho "power: ..." is enough.